### PR TITLE
Js/javascript1/week3

### PR DESCRIPTION
**Pull request by qamarfj**

### CODE REVIEW

File: javascript/javascript1/week3/index.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    const newActivity = {
+        date: getDate(),
+        activity: activity,
+        duration: duration,
+    };
+    activities.push(newActivity);
+}
+
+/**return current date in string format i.e. dd/mm-yyyy */
+function getDate() {
```
Comment (aina21) [inline javascript/javascript1/week3/index.js:109]: nice :+1:

**Review by aina21 — APPROVED** [review]
well done! :+1:

**Review by aina21 — CHANGES_REQUESTED** [review]
you missed Note-taking app exercise.

**Comment by qamarfj** [comment]
> you missed Note-taking app exercise.

Sorry I forgot to add file but Now I  have uploaded Note taking app exercise

File: javascript/javascript1/week3/index.js (hunk 1 above)
Comment (qamarfj) [reply javascript/javascript1/week3/index.js:109]: Thank you and Note taking app exercise added

**Review by aina21 — APPROVED** [review]
well done!

**Pull request merged**